mysql/datadog_checks/mysql/innodb_metrics.py

```diff
@@ -0,0 +1,404 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+import logging
+import re
+from collections import defaultdict
+from contextlib import closing
+
+import pymysql
+from six import PY3, iteritems
+
+from datadog_checks.base import is_affirmative
+
+from .collection_utils import collect_scalar
+from .const import OPTIONAL_INNODB_VARS
+
+if PY3:
+    long = int
+
+log = logging.getLogger(__name__)
```

Comment:
The issue with using a local logger is that we don't have check instance id which can be useful for investigation/support.

Could you use a class to contain `get_stats_from_innodb_status` and then use `self.log = get_check_logger()` https://github.com/DataDog/integrations-core/blob/f883e0a14a7b3bfa91cb7d3ae89c5a896af9504b/datadog_checks_base/tests/test_log.py#L43-L56

note: Using `get_check_logger` directly in a function is not recommended since `get_check_logger` frame lookup have a cost (I didn't measure that yet, but likely better to avoid too many lookups it if possible).